public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/AddAnomalyDetector.tsx

```diff
@@ -231,6 +286,80 @@ function AddAnomalyDetector({ embeddable, closeFlyout, mode, setMode }) {
     }
   };
 
+  const getEverythingSuccessfulButton = (detectorId, shingleSize) => {
+    return (
+      <EuiText>
+        <p>
+          Attempting to initialize the detector with historical data. This
+          initializing process takes approximately 1 minute if you have data in
+          each of the last {32 + shingleSize} consecutive intervals.
+        </p>
+        {alertingExists(detectorId) ? (
+          <EuiFlexGroup>
+            <EuiFlexItem>
+              <p>Set up alerts to be notifified of any anomalies.</p>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <div>
+                <EuiButton onClick={() => openAlerting(detectorId)}>
+                  Set up alerts
+                </EuiButton>{' '}
+              </div>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        ) : null}
+      </EuiText>
+    );
+  };
+
+  const alertingExists = () => {
+    try {
```

Comment:
we have a setup alert button on existing AD detector overview page, is this how we currently check if alerting exist? also what does `ALERTING_TRIGGER_AD_ID` mean here?

Reply:
We actually don't have this logic on the setup alert button in AD overview page. If there is no alerting plugin, the button still shows and it just sends to a broken page.
`ALERTING_TRIGGER_AD_ID` is the trigger that David set up on the alerting code to open up the relevant flyout. If the alerting trigger doesn't exist then it means there is no alerting plugin or potentially something else going on but regardless it means the flyout for creating a monitor on top of a detector wont be opened

Reply:
> We actually don't have this logic on the setup alert button in AD overview page. If there is no alerting plugin, the button still shows and it just sends to a broken page.

I believe we used to have a check here. Regardless, this is bad UX. Can you open an issue on that. We can probably keep it broad, like "clean up how to detect alerting plugin is installed"